app/Console/Commands/AuditDependencies.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class AuditDependencies extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'dependencies:audit';
```

Comment:
Any reason why this PHPDoc comment can't be converted to a native PHP type declaration?  PHPDoc comments aren't analyzed by PHPStan so there's no guarantee that they are correct.  Apply throughout.

Reply:
Somehow, there is a reason, yes.  When I change the line to 
```
    /**
     * The name and signature of the console command.
     * @var string
     */
    protected string $signature = 'dependencies:audit';
```
I get the following error when `build`-ing the `common` image.

```
> @php artisan package:discover --ansi
PHP Fatal error:  Type of App\Console\Commands\AuditDependencies::$signature must not be defined (as in class Illuminate\Console\Command) in /home/kitware/cdash/app/Console/Commands/AuditDependencies.php on line 7

In AuditDependencies.php line 7:
                                                                               
  Type of App\Console\Commands\AuditDependencies::$signature must not be defi  
  ned (as in class Illuminate\Console\Command)  
  ```